.pullapprove.yml

```diff
@@ -160,7 +164,7 @@ groups:
       request: 2
     reviewers:
       users:
-        - ldgauthier # Laura Gauthier
+        - samuelklee # Samuel Lee
```

Comment:
I'm happy to help review any changes to these pipelines, but strictly speaking, I think Laura only handed off responsibilities for the joint-genotyping pipelines to me. Perhaps @kachulis can clarify who the primary owner for the single-sample pipelines will be going forward?

Reply:
(Or if it turns out that I do own these now, it would be good to sort that out!)

Reply:
OK---chatted with Chris and I understand the intended split now. He'll own everything up through alignment/duplicate marking.

Reply:
thanks sam! that sounds good 👍